src/popper/scm.py

```diff
@@ -167,10 +167,16 @@ def get_sha(repo, short=None):
     if not repo:
         return None
 
-    if short:
-        return repo.git.rev_parse(repo.head.object.hexsha, short=short)
+    try:
+        if short:
+            sha = repo.git.rev_parse(repo.head.object.hexsha, short=short)
+        else:
+            sha = repo.git.rev_parse(repo.head.object.hexsha)
+
+    except ValueError as e:
```

Comment:
I was thinking about this and was wondering whether it'd be more friendly to show a warning instead of failing. Letting the user know that git information is ignored because the repository doesn't have a commit. We show the message and then return `None`, what do you think?

Reply:
Yeah, I think that should be fine. Thanks